### ISSUE TITLE
question about data use

### ISSUE THREAD

**Issue by cdhx** [wenhuchen/KGPT#8]
hi, thanks for publish this great work!

i have some question about data use

1. in test set of web nlg, there are some example that do not have a text label, how did you evaluate them
2. webnlg have multi text label for a triple set(for example: triple_set1 can map to [text1,text2,text3]), during training, did you regard one example as multi training data(for above example, we got 3 training data :[(triple_set1,text1) ,(triple_set1,text2),(triple_set1,text3)]) or randomly chose one of the text label for one example? if a triple set have multi text label during training, will the model get confused because the model learn   triple_set1 map to text1, but when meet the example triple_set1 map to text2, it will lose some bleu score and it will swing between text1 and text2. what is the standard evaluate way in this task(I am new to this area, sorry for some stupid question).

looking forward to your replay, any help would be appreciated!
